From 45081d6ff61e0f12c1c53545e3bd1b5aed3100b4 Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Mon, 17 Sep 2018 01:32:20 +0000
Subject: [PATCH 14/18] test/sysroot-crates-are-unstable: Fix test when rpath
 is disabled

Without this environment var, the test can't run rustc to find
the sysroot path.
---
 .../run-make-fulldeps/sysroot-crates-are-unstable/Makefile    | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile b/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile
index a35174b3c2a..9e770706857 100644
--- a/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile
+++ b/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile
@@ -1,2 +1,4 @@
+-include ../tools.mk
+
 all:
-	python2.7 test.py
+	env '$(HOST_RPATH_ENV)' python2.7 test.py
-- 
2.21.0

